oaknational/Oak-Web-Application
# feat: add support for presentational-only images

## Description
- Adds support for ticking `isPresentational` in the CMS to override alt text

As for exactly how presentational images are to be marked up, you may all fight amongst yourself and either update the PR or let me know what attributes are desired

## Issue(s)

Fixes #647 

## How to test

1. Go to the dummy testing [blog post in the CMS](https://oak.sanity.studio/desk/webinarsBlogs;blogPosts;2ed3149d-8c7d-41cb-9123-12ff405ff979)
2. Set "is presentational" to true on an image in the blog - and add some alt text (this will warn it's invalid, but persevere)
3. Go to the PR preview with preview mode enabled
https://oak-web-application-git-feat-presen-3a9885-oak-national-academy.vercel.app/api/preview/blog/some-blog-post?secret=LcLt4fj87uUitpZJ
4. The alt text you entered should not be read out (or in the DOM)

## Screenshots

![Google Chrome - 2022-09-07 at 16 38 04](https://user-images.githubusercontent.com/2717635/188920125-210a4699-6a15-4d37-bb3b-963ce07c1db7.png)


## Checklist
